Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,627 @@
+
+Much of the process of preparing a machine to join an active domain is a matter of preparing. IF the server and client 
+machine is already set up right, then the process of joining usually goes very smoothly. If there are problems in the setup, 
+then debugging becomes difficult. This document will try to call out the things to check in advance.
+
+Due to the limitations of netbios which are still reflected in the system, it is a good idea to limit hostnames to 15 characters.
+
+Microsoft recommends this, and we have had some rather unpleasant issues around violating that rule.  The reason is that 
+the netbios host name (referred to in the domain controllers data as SAM is based on the netbios name. Deviation between the shortname and the SAM name can cause problems. It just isnt worth it.
+
+A link here.(https://technet.microsoft.com/en-us/library/cc726016(v=ws.10).aspx)
+
+  - The actual creation of computer and user accounts is done via the "Active Domain Users And Computers" utility on the ADC.
+  - Service Principals are created using the setspn.exe command line utility. 
+  - Information about the active domain state can be gotten from via ldapsearch (Windows or Cleint side) or throgh powershell,
+    via the get-ADComputer() and get-ADUser cmdlets.
+
+### Redhat/Centos Linux 7 (all versions)  (Should also apply to Fedora 24 and later).
+
+Redhat has an excellent write up on setting up an active domain client 
+here.
+(https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Linux_Domain_Identity_Authentication_and_Policy_Guide/index.html)
+which describes the process with all of the possible considerations.  Some might find it a bit long, but it is recommended
+reading.
+
+####  Required Packages:
+
+We recommend you use realmd to join. The required packages in that care are 
+```
+    sudo yum install sssd sssd-client krb5-workstation samba openldap-clients open-ssl authconfig realmd oddjob oddjob-mkhomedir adcli 
+```
+
+#### Setup after package install
+
+
+Tasks:
+
+  - Hostname - edit /etc/hostname to create an fqdn with a short name (the hostname proper) of 15 characters or less.
+
+  - DNS - point at the ADC as the dns name server by editing the source /etc/resolv.conf. Optionally add an entry in /etc/hosts.
+    pointing at the ADC to ensure that is available.
+
+    Note:  If you are using Network Manager, just modifying /etc/resolv.conf will not me sufficient in the long run as 
+    Network Manager clobbers /etc/resolv.conf on every reboot. If so, modify the network manager configuration files 
+    using the redhat-network-configuration tool or as needed. 
+    *but ensure you have a fully qualified domain name by default before proceeding*
+
+  - NTP is necessary. It should point at the ADC plus a number of pool servers. Authentication will fail more or less 
+    mysteriously if the time is out of sync by more than five minutes between the kdc, client and host its trying to 
```

Comment:
get rid of "more or less mysteriously", it doesn't add anything. It fails if time is out of sync, fine.